cmake/Makefile.tests.common

```diff
@@ -0,0 +1,94 @@
+# This file contains common targets for building and testing tests, examples and benchmarks
```

Comment:
Could add this file to .gitattributes for highlighting.

Reply:
Done.